providers/constants/oracle.go

```diff
@@ -0,0 +1,62 @@
+package constants
+
+import (
+	"math/big"
+
+	apihandlers "github.com/skip-mev/slinky/providers/base/api/handlers"
+	providertypes "github.com/skip-mev/slinky/providers/types"
+	mmtypes "github.com/skip-mev/slinky/x/marketmap/types"
+)
+
+type (
+	// PriceAPIQueryHandler is a type alias for the API query handler that is used to fetch price data
+	// from the price providers.
+	PriceAPIQueryHandler = apihandlers.APIQueryHandler[mmtypes.Ticker, *big.Int]
+
+	// PriceAPIDataHandler is a type alias for the API data handler that is used to fetch resolve http
+	// requests and parse the response into price data.
+	PriceAPIDataHandler = apihandlers.APIDataHandler[mmtypes.Ticker, *big.Int]
+
+	// PriceResponse is a type alias for the response from the price providers.
+	PriceResponse = providertypes.GetResponse[mmtypes.Ticker, *big.Int]
+)
+
+var (
+	// The following are the default tickers for the oracle sidecar.
+	//
+	// USD denominated tickers.
+	ATOM_USD     = mmtypes.NewTicker("ATOM", "USD", 8, 1)
+	AVAX_USD     = mmtypes.NewTicker("AVAX", "USD", 8, 1)
+	BITCOIN_USD  = mmtypes.NewTicker("BITCOIN", "USD", 8, 1)
```

Comment:
Some feedback we've gotten from Initia is that they currently have `ETHEREUM` and `BITCOIN` on chain and want `ETH` `BTC`. I guess we should just make a conscious decision to support 1 or the other in terms of long/short forms and not have both. Because right now we have short forms on Berachain and long on Initia.

I'm fine w/ long form names, but then you'll want to standardize things like `AVALANCHE` instead of `AVAX`.